tsl/test/sql/include/cagg_bgw_common.sql

```diff
@@ -56,7 +56,11 @@ CREATE TABLE public.bgw_log(
 );
 
 CREATE VIEW sorted_bgw_log AS
-    SELECT * FROM bgw_log ORDER BY mock_time, application_name COLLATE "C", msg_no;
+    SELECT msg_no,
+    	   mock_time,
+    	   application_name,
+    	   regexp_replace(regexp_replace(msg, '(Wait until|started at|execution time) [0-9]+(\.[0-9]+)?', '\1 (RANDOM)', 'g'), 'background worker "[^"]+"','connection') AS msg
```

Comment:
Nit: We do similar replacements in [the test runner](https://github.com/timescale/timescaledb/blob/36e71000132cf3a5430849c11ff99e910ef81207/test/runner_shared.sh#L81). It could be beneficial to do it there since it would apply to all tests.

Reply:
Didn't do that here since it is a quite specific replacement for these tests.